components/lvgl.rst

```diff
@@ -0,0 +1,2449 @@
+
+- **hidden** (*Optional*, boolean): make the widget hidden (like it wasn't there at all), also usable with :ref:`shorthand <lvgl-objupd-shorthands>` actions ``lvgl.widget.show`` and ``lvgl.widget.hide``. Hidden objects are ignored in layout calculations. Defaults to ``false``.
+- **checkable** (*Optional*, boolean): toggle checked state when the widget is clicked.
+- **clickable** (*Optional*, boolean): make the widget clickable by input devices. Defaults to ``true``. If ``false``, it will pass the click to the widgets behind it (clicking through).
+- **scrollable** (*Optional*, boolean): the widget can become scrollable. Defaults to ``true`` (also see the ``scrollbar_mode`` property).
+- **scroll_elastic** (*Optional*, boolean): allow scrolling inside but with slower speed.
+- **scroll_momentum** (*Optional*, boolean): make the widget scroll further when "thrown".
+- **scroll_one** (*Optional*, boolean): allow scrolling only on ``snappable`` children.
+- **scroll_chain_hor** (*Optional*, boolean): allow propagating the horizontal scroll to a parent.
+- **scroll_chain_ver** (*Optional*, boolean): allow propagating the vertical scroll to a parent.
+- **scroll_chain simple** (*Optional*, boolean): packaging for (``scroll_chain_hor | scroll_chain_ver``).
+- **scroll_on_focus** (*Optional*, boolean): automatically scroll widget to make it visible when focused.
+- **scroll_with_arrow** (*Optional*, boolean): allow scrolling the focused widget with arrow keys.
+- **click_focusable** (*Optional*, boolean): add focused state to the widget when clicked.
+- **snappable** (*Optional*, boolean): if scroll snap is enabled on the parent it can snap to this widget.
+- **press_lock** (*Optional*, boolean): keep the widget pressed even if the press slid from the widget.
+- **event_bubble** (*Optional*, boolean): propagate the events to the parent.
+- **gesture_bubble** (*Optional*, boolean): propagate the gestures to the parent.
+- **adv_hittest** (*Optional*, boolean): allow performing more accurate hit (click) test. For example, may help by accounting for rounded corners.
+- **ignore_layout** (*Optional*, boolean): the widget is simply ignored by the layouts. Its coordinates can be set as usual.
+- **floating** (*Optional*, boolean): do not scroll the widget when the parent scrolls and ignore layout.
+- **overflow_visible** (*Optional*, boolean): do not clip the children's content to the parent's boundary.
+- **layout_1**, **layout_2** (*Optional*, boolean): custom flags, free to use by layouts.
+- **widget_1**, **widget_2** (*Optional*, boolean): custom flags, free to use by widget.
+- **user_1**, **user_2**, **user_3**, **user_4** (*Optional*, boolean): custom flags, free to use by user.
+
+.. note::
+
+    LVGL only supports **integers** for numeric ``value``. Visualizer widgets can't display floats directly, but they allow scaling by 10s.
+
+.. _lvgl-wgt-lbl:
+
+``label``
```

Comment:
I think widgets should be in alphabetical order

Reply:
The order is deliberately not alphabetical. It's from the simplest to the most complex. Label and text are the simplest ones.

Otherwise it would start with `arc` which is by itself a pretty complex and rather seldom used widget.